client/js/components/menu.jsx

```diff
@@ -4,9 +4,18 @@ var $ = require("jquery");
 var Menu = React.createClass({
   componentDidMount: function() {
     var DROP_DOWN_SPEED = 300;
+    var TOGGLED = false;
```

Comment:
all caps is generally reserved for constants. since you're reassigning the var, camel case is recommended.